website/docs/docs/dbt-semantic-layer.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "dbt Semantic Layer"
+id: dbt-semantic-layer
+description: "Introducing the dbt Semantic Layer"
+sidebar_label: "dbt Semantic Layer"
+---
+
+:::info Coming soon
+The dbt Semantic Layer is undergoing some sophisticated changes, enabling more complex metric definitions and efficient querying. As part of these changes, the dbt_metrics package will be deprecated and replaced with MetricFlow. For more info, check out the [The dbt Semantic Layer: what's next?](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) and [dbt_metrics deprecation](https://docs.getdbt.com/blog/deprecating-dbt-metrics) blog.
```

Comment:
do we want to link to the new 1.6 beta 'build your sl' docs from here? i left it out because it may not necessarily apply to this audience, however it may also be a good opportunity to help users transition.